### DISCUSSION TITLE
How to enable desktop mode?

### DISCUSSION THREAD

**Discussion by 0rangutan** [Archfx/duo-de#28] [Q&A] [answered]
@Archfx - Many thanks for putting this ROM together.
I followed your instructions to dirty flash direct from 14 Pixel Experience, however desktop mode does not seem to be working, even though enabled via Home Settings.  I've tried toggling this on and off with no result.
Any suggestions?  Thanks!

**Comment by Archfx** [comment, marked as answer]
Please enable the following settings from the developer options.

- Force activities to be resizable
- Enable freeform windows
- Enable non-resizable in multi-window

It looks like vendor overrides for floating windows are not applied during the build to enable it by default. I will fix this in the next release

**Comment by 0rangutan** [comment]
That fixed it, thanks

**Comment by sdny10000** [comment]
This is a great rom, thank you for sharing! I had the desktop mode enabled. Now I tried to disable it. But the floating windows cannot be turned off. Tried to disable "Enable freeform window" option, but I am not able to disable it. It always shows on.

**Reply by Archfx** [reply]
Thanks, above steps are one time setting, which you don't have to do it on latest releases.

 In order to disable desktop mode, please use the `home settings` (hold on your desktop until popup menu comes up) and remove the tick from 'enable desktop mode' as stated in the readme. This should disable the desktop mode.

**Reply by sdny10000** [reply]
i see, excellent! thank you!
